teerex/src/lib.rs

```diff
@@ -76,7 +76,7 @@ pub mod pallet {
 		Forwarded(ShardIdentifier),
 		ShieldFunds(Vec<u8>),
 		UnshieldedFunds(T::AccountId),
-		ProcessedParentchainBlock(T::AccountId, H256, H256),
+		ProcessedParentchainBlock(T::AccountId, H256, H256, u64),
```

Comment:
Is it not possible to use the runtime Blocknumber? E.g. T::BlockNumber?

Reply:
Should be possible, but this would require a change in the extrinsic signature too, so I guess, we leave it as it is.

Reply:
Would it? Either way: Use u32 instead of u64. https://github.com/integritee-network/parachain/blob/master/polkadot-parachains/integritee-runtime/src/lib.rs#L751-L752

Reply:
@gaudenzkessler told me that the signature change is required anyhow, so I think we should change it.